IngaChen_ingachen/hw_3/Book/Gemfile

```diff
@@ -0,0 +1,40 @@
+source 'https://rubygems.org'
+
+
+# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
+gem 'rails', '4.1.6'
+# Use sqlite3 as the database for Active Record
+gem 'sqlite3'
+# Use SCSS for stylesheets
+gem 'sass-rails', '~> 4.0.3'
+# Use Uglifier as compressor for JavaScript assets
+gem 'uglifier', '>= 1.3.0'
+# Use CoffeeScript for .js.coffee assets and views
+gem 'coffee-rails', '~> 4.0.0'
+# See https://github.com/sstephenson/execjs#readme for more supported runtimes
+gem 'therubyracer',  platforms: :ruby
+
+# Use jquery as the JavaScript library
+gem 'jquery-rails'
+# Turbolinks makes following links in your web application faster. Read more: https://github.com/rails/turbolinks
+gem 'turbolinks'
+# Build JSON APIs with ease. Read more: https://github.com/rails/jbuilder
+gem 'jbuilder', '~> 2.0'
+# bundle exec rake doc:rails generates the API under doc/api.
+gem 'sdoc', '~> 0.4.0',          group: :doc
+
+# Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
+gem 'spring',        group: :development
+
+# Use ActiveModel has_secure_password
+# gem 'bcrypt', '~> 3.1.7'
+
```

Comment:
Hm I'm not sure actually. I didn't think I touched those.

On Tue, Oct 27, 2015 at 6:38 PM, Nicole Borgaard <notifications@github.com>
wrote:

> In IngaChen_ingachen/hw_3/Book/Gemfile
> <https://github.com/Borgaard/bewd11-homework/pull/30#discussion_r43210834>
> :
>
> > +
> > +# Use jquery as the JavaScript library
> > +gem 'jquery-rails'
> > +# Turbolinks makes following links in your web application faster. Read more: https://github.com/rails/turbolinks
> > +gem 'turbolinks'
> > +# Build JSON APIs with ease. Read more: https://github.com/rails/jbuilder
> > +gem 'jbuilder', '~> 2.0'
> > +# bundle exec rake doc:rails generates the API under doc/api.
> > +gem 'sdoc', '~> 0.4.0',          group: :doc
> > +
> > +# Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
> > +gem 'spring',        group: :development
> > +
> > +# Use ActiveModel has_secure_password
> > +# gem 'bcrypt', '~> 3.1.7'
> > +
>
> Any reason the development and test gem groups were altered?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Borgaard/bewd11-homework/pull/30/files#r43210834>.
>